test/integration_test.py

```diff
@@ -2045,6 +2046,9 @@ def test_response_security_headers(self):
             '/oauth2_redirect': {'Cache-Control': 'no-store'},
             '/health/basic': {'Cache-Control': 'no-store'}
         }
+        # https://www.w3.org/TR/CSP2/#policy-syntax
+        directive_re = re.compile(r'\s*([a-zA-Z0-9-]+)((?:\s+[^\s;,]*)*)')
```

Comment:
I believe this is wrong. The grammar requires exactly one space between name and value. You're using `+` (one or more). Also, that separating space is included in the second capturing group, the value. It shouldn't because the space is not part of the value.